ui/pages/confirmations/send/send-content/__snapshots__/send-content.component.test.js.snap

```diff
@@ -1,5 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`SendContent Component SendHexDataRow should not render the SendHexDataRow if props.showHexData is false 1`] = `<div />`;
```

Comment:
Is this change intentional?

Reply:
no, this seems to be that bug we are seeing where local and CI expect different results for this component's snapshot